crates/red_knot_test/src/parser.rs

```diff
@@ -618,7 +622,7 @@ impl<'s> Parser<'s> {
                 EmbeddedFilePath::Explicit(path)
             }
             None => match lang {
-                "py" => EmbeddedFilePath::Autogenerated(PySourceType::Python),
+                "py" | "python" => EmbeddedFilePath::Autogenerated(PySourceType::Python),
```

Comment:
Not that it really matters much, but one reason why it is quite nice to stick with only using ```` ```py ```` or ```` ```pyi ```` is that currently all the tags we use exactly map to the file extension that the file will be saved as in the `MemorFileSystem`. If you use ```` ```py ````, it saves it with the `.py` file extension; if you use ```` ```pyi ````, it saves it with the `.pyi` file extension; if you use ```` ```toml ````, it saves it as a `.toml` file... but ```` ```python ```` here isn't going to cause mdtest to save the snippet with a `.python` extension.

Reply:
Stop looking at my PR, it's in draft ;)

I was told yesterday that this wouldn't be hard to change but I precisely haven't changed it before because I was afraid of the push back and discussions it involves. 

I can revert the change here and open a separate PR but I'm very used to writing `python` and `RustRover` suggestions `python` but never `py`, making it very annoying because tests always fail.


I'd find this a quality-of-life improvement. I agree, it leads to less consistency but it's common that markdown editors/renderers support both: names and extension.